src/c-writer.cc

```diff
@@ -1654,28 +1672,43 @@ void CWriter::Write(const BinaryExpr& expr) {
       break;
 
     case Opcode::I32Rotl:
+      WritePrefixBinaryExpr(expr.opcode, "I32_ROTL");
+      break;
+
     case Opcode::I64Rotl:
-      UNIMPLEMENTED(expr.opcode.GetName());
+      WritePrefixBinaryExpr(expr.opcode, "I64_ROTL");
       break;
 
     case Opcode::I32Rotr:
+      WritePrefixBinaryExpr(expr.opcode, "I32_ROTR");
+      break;
+
     case Opcode::I64Rotr:
-      UNIMPLEMENTED(expr.opcode.GetName());
+      WritePrefixBinaryExpr(expr.opcode, "I64_ROTR");
       break;
 
     case Opcode::F32Min:
+      WritePrefixBinaryExpr(expr.opcode, "fminf");
+      break;
+
     case Opcode::F64Min:
-      UNIMPLEMENTED(expr.opcode.GetName());
+      WritePrefixBinaryExpr(expr.opcode, "fmin");
```

Comment:
Didn't notice this! Looks like it has subtly different behavior w/ NaNs though:

```
fmin(3.0, NAN) => 3.0
(f32.min (f32.const 3.0) (f32.const nan)) => nan
```

From looking at http://en.cppreference.com/w/cpp/numeric/math/fmin.